pkg/ctl/utils/describe_stacks.go

```diff
@@ -29,20 +46,29 @@ func describeStacksCmd(cmd *cmdutils.Cmd) {
 		fs.BoolVar(&all, "all", false, "include deleted stacks")
 		fs.BoolVar(&events, "events", false, "include stack events")
 		fs.BoolVar(&trail, "trail", false, "lookup CloudTrail events for the cluster")
+		fs.StringVarP(&output, "output", "o", "", "specifies the output formats (valid option: json and yaml)")
 		cmdutils.AddTimeoutFlag(fs, &cmd.ProviderConfig.WaitTimeout)
 	})
 
 	cmdutils.AddCommonFlagsForAWS(cmd.FlagSetGroup, &cmd.ProviderConfig, false)
 }
 
-func doDescribeStacksCmd(cmd *cmdutils.Cmd, all, events, trail bool) error {
+func doDescribeStacksCmd(cmd *cmdutils.Cmd, all, events, trail bool, printer printers.OutputPrinter, output printers.Type) error {
+	if output != "" && (all || events || trail) {
+		logger.Warning("Since the output flag is specified, the flags `all`, `events` and `trail` (if set) will not be used.")
+		cmd.CobraCommand.DisableFlagParsing = true
+		os.Exit(3)
```

Comment:
`os.Exit` is not required here, just returning an error will handle exiting the command with a non-zero status.
 
(Also, as mentioned above, this should be in `RunE`)